helloWorld.py

```diff
@@ -0,0 +1,11 @@
+### What is this, some sort of test file? :O
+
+greeting = "Hello World"
+
+for letter in greeting:
+	print(letter, end='')
```

Comment:
This is inspired, I spent weeks on this problem and did not come up with a solution half as elegant. :shipit:  🚀